lib/rate_limiter.rb

```diff
@@ -79,13 +80,38 @@ def can_perform?
     PERFORM_LUA_SHA = Digest::SHA1.hexdigest(PERFORM_LUA)
   end
 
+  unless defined? PERFORM_LUA_AGGRESSIVE
```

Comment:
I don't think this is required since each file should only be loaded once.

Reply:
yeah I am not sure about this, just followed the same pattern as above. we can try without I guess is a separate PR